src/api.rs

```diff
@@ -1457,50 +1457,6 @@ impl Api {
         Ok(rv)
     }
 
-    /// Create a new checkin for a monitor
-    pub fn create_monitor_checkin(
```

Comment:
These APIs are deprecated in sentry and should not be used in favor of just sending check-ins as envelopes to relay.